bam.lua

```diff
@@ -391,8 +391,8 @@ function BuildContent(settings, arch, conf)
 		end
 		-- dependencies
 		dl = Python("scripts/download.py")
-		AddJob("other/sdl/include/SDL.h", "Downloading SDL2", dl .. " sdl")
-		AddJob("other/freetype/include/ft2build.h", "Downloading freetype", dl .. " freetype")
+		AddJob({"other/sdl/include/SDL.h", "other/sdl/windows/lib64/SDL2.dll"}, "Downloading SDL2", dl .. " sdl")
```

Comment:
Does that work when building 32bit?
Shouldn't it consider the arch you build for `"other/sdl/windows/lib" .. _arch .. "/SDL2.dll"`?

Reply:
Not sure how I missed that, thanks